src/components/SearchBar/searchBarChangelogData.ts

```diff
@@ -9,6 +9,15 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Styles"],
+    notes: [
+      "Updates Searchbar style to remove stacking on mobile breakpoints, adds span element to text for CSS targeting.",
```

Comment:
This is a little misleading because the component will still "stack" when the select element is enabled.  Just update this statement to mirror what is in the CHANGELOG.